great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+	  <header>
+ 	<section class="section1">
+	  <nav>
+	 Services
```

Comment:
This works, but try to making each of the `nav` elements anchor elements. That should help you practice overriding default browser styles with your own CSS, and better align these elements with their intended function.